opp_msgs_serialization/include/opp_msgs_serialization/opp_msgs_yaml.h

```diff
@@ -99,14 +99,13 @@ struct convert<noether_msgs::SurfaceWalkRasterGeneratorConfig>
     node["min_hole_size"] = rhs.min_hole_size;
     node["min_segment_size"] = rhs.min_segment_size;
     node["raster_rot_offset"] = rhs.raster_rot_offset;
-    node["raster_wrt_global_axes"] = rhs.raster_wrt_global_axes;
     node["generate_extra_rasters"] = rhs.generate_extra_rasters;
     return node;
   }
 
   static bool decode(const Node& node, noether_msgs::SurfaceWalkRasterGeneratorConfig& rhs)
   {
-    if (node.size() != 9)
+    if (node.size() != 10)
```

Comment:
I think this should be 8 instead of 10 since `raster_wrt_global_axes` has been removed

```suggestion
    if (node.size() != 8)
```